src/tests/tribol_common_plane_penalty.cpp

```diff
@@ -31,38 +33,32 @@
 #include <iomanip>
 #include <fstream>
 
-using real = tribol::real;
+using RealT = tribol::RealT;
 
 void compareGaps( tribol::CouplingScheme const * cs, 
-                  real gap, const real tol,
+                  RealT gap, const RealT tol,
                   const char *gapType )
 {
-   tribol::ContactPlaneManager& cpManager = tribol::ContactPlaneManager::getInstance();
-   tribol::InterfacePairs const * const pairs = cs->getInterfacePairs();
-   tribol::IndexType const numPairs = pairs->getNumPairs();
+   tribol::IndexT const numPairs = cs->getNumActivePairs();
+   // TODO: get rid of the const cast if we can
+   const auto cs_view = const_cast<tribol::CouplingScheme*>(cs)->getView();
```

Comment:
This isn't a const cs viewer.  We need a non-const version of the coupling scheme to get a view since it can potentially change the values it holds (even if we don't in this routine).  We can look at re-introducing the const View for cases like this down the line, but we'll leave this as is for simplicity.